lumigator/jobs/inference/schemas.py

```diff
@@ -11,6 +13,12 @@ class DatasetConfig(BaseModel):
     model_config = ConfigDict(extra="forbid")
 
 
+class TaskType(str, Enum):
+    SUMMARIZATION = "summarization"
+    TRANSLATION = "translation"
+    TEXT_GENERATION = "text-generation"
```

Comment:
This might be a slight aside from your PR, but where do we use TEXT_GENERATION as a task? It's not clear to me why this is marked as a task, since currently Lumigator supports summarization as a task and we're supporting translation next, but text-generation we didn't add yet, right?

Reply:
Cool! Gets me thinking, do you think we can then tell the UI/Frontend people that they can add support for text-generation as a task?